src/app/lib/views/file_selector.js

```diff
@@ -74,10 +74,9 @@
             var torrent = _this.model.get('torrent');
             var file = parseInt($(e.currentTarget).attr('data-file'));
             var actualIndex = parseInt($(e.currentTarget).attr('data-index'));
-            torrent.name = torrent.files[file].name;
 
             var torrentStart = new Backbone.Model({
-                torrent: torrent,
+                torrent: torrent.magnetURI,
```

Comment:
paste a link from http i.e. put a file on a webserver and throw the URL on Butter